worlds/witness/player_logic.py

```diff
@@ -186,7 +196,9 @@ def make_single_adjustment(self, adj_type, line):
 
             if len(line_split) > 2:
                 required_items = parse_lambda(line_split[2])
-                items_actually_in_the_game = {item[0] for item in StaticWitnessLogic.ALL_SYMBOL_ITEMS}
+                items_actually_in_the_game = [item_name for item_name, item_definition
+                                              in StaticWitnessLogic.all_items.items()
+                                              if item_definition.category is ItemCategory.SYMBOL]
```

Comment:
I like this a lot btw, the fact you made an enum for ItemCategory and pushed all of the items into a single list

Reply:
i love list comprehensions, i wish they were in every language and that people would not get mad at me for using them in inappropriate places